addons/message_plan_upgrade_87d copy/conditions.js

```diff
@@ -0,0 +1,34 @@
+(function(api, condition) {  
+  //Show this message after 87 days
+
+  //There are 2 identical (content-wise), yet separate messages because we want to give the illusion that we are 
+  //re-showing the same message 87 days in (by hiding the 14 day one and showing the 87 day one) without having to
+  //un-read or un-dismiss a message
+
+  api.connectSignal(api.subscriptionData, 'changed', () => computeCondition());
+
+  //Slight delay to as we wait for subscriptionData to initialize
+  api.setTimeout(5000, () => computeCondition());
```

Comment:
Is this considered reliable to ensure that TaskGetSubscriptionDetails has run? Or should that be explicitly checked?

Reply:
I would say no, and I am looking for a better solution. We aren't really waiting on `TaskGetSubscriptionDetails` (if that task runs, and we have new subscription data, the `connectSignal` above will catch it), we're waiting on the subscriptionData from settings to be loaded into memory

Reply:
I think adding `emit changed();` [here](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/e1c86b0063dbe83dbbc5b809a03fcd3eee8d14be/src/models/subscriptiondata.cpp#L59) could resolve the issue